arm-sql/2015-05-01-preview/swagger/failoverGroups.json

```diff
@@ -23,178 +23,227 @@
           }
         },
-        "deprecated": false
+        "x-ms-examples": {
+          "Get failover group": {
+            "$ref": "../examples/FailoverGroupGet.json"
+          }
+        }
       },
       "put": {
         "tags": [
           "FailoverGroups"
         ],
         "description": "Creates or updates a failover group.",
         "operationId": "FailoverGroups_CreateOrUpdate",
-        "x-ms-examples": {
-          "Create failover group": {
-            "$ref": "../examples/FailoverGroupCreateOrUpdate.json"
-          }
-        },
         "parameters": [
           {
-            "$ref": "#/parameters/ApiVersionParameter"
```

Comment:
nit: maintain a consistent parameter ordering, either place ApiVersion and SubscriptionId together at the beginning of the parameters section or at the end.